cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -453,6 +482,12 @@ instance ToObject (WithDomainName DnsTrace) where
              , "domain" .= show dom
              , "event" .= show ev ]
 
+instance Show ptcl => ToObject (WithMuxBearer (MuxTrace ptcl)) where
+  toObject _verb (WithMuxBearer b ev) =
+    mkObject [ "kind" .= String "MuxTrace"
+             , "bearer" .= show b
+             , "event" .= show ev ]
```

Comment:
this could probably be extended later to provide more details for the events. OK for now.